turbo.json

```diff
@@ -8,36 +8,65 @@
       "cache": true
     },
     "build": {
+      "dependsOn": [
+        "generate"
+      ]
+    },
+    "test": {},
+    "test:unit": {},
+    "test:integration": {},
+    "test:watch": {},
+    "lint": {},
+    "lint:fix": {},
+    "dev": {
+      "cache": false,
+      "persistent": true
+    },
+    "clean": {
+      "cache": false
+    },
+    "@zitadel/server#generate": {
+      "outputs": [
+        "src/proto/**"
+      ]
+    },
+    "@zitadel/client#generate": {
+      "outputs": [
+        "src/proto/**"
+      ]
+    },
+    "@zitadel/login#build": {
       "outputs": [
         "dist/**",
         ".next/**",
         "!.next/cache/**"
       ],
       "dependsOn": [
-        "lint",
-        "generate",
         "^build"
       ]
     },
-    "test": {
+    "@zitadel/login#test": {
       "dependsOn": [
-        "generate",
-        "@zitadel/server#build"
+        "@zitadel/server#build",
+        "@zitadel/react#build"
       ]
     },
-    "test:watch": {
+    "@zitadel/login#test:integration": {
       "dependsOn": [
-        "generate",
-        "@zitadel/server#build"
+        "@zitadel/server#build",
+        "@zitadel/react#build"
       ]
     },
-    "lint": {},
-    "dev": {
-      "cache": false,
-      "persistent": true
+    "@zitadel/login#test:unit": {
+      "dependsOn": [
+        "@zitadel/server#build"
+      ]
     },
-    "clean": {
-      "cache": false
+    "@zitadel/login#test:watch": {
+      "dependsOn": [
+        "@zitadel/server#build",
+        "@zitadel/react#build"
+      ]
     }
```

Comment:
Let's please move this config to package specific turbo.json files in @zitadel/server, @zitadel/client and @zitadel/login :pray:

Reply:
ok we can move it to the packages themselves 👍

Reply:
Done